src/ifp/test/init_floorplan1.py

```diff
@@ -0,0 +1,19 @@
+from openroad import Tech, Design
+import helpers
+
+tech = Tech()
+tech.readLEF("Nangate45/Nangate45_tech.lef")
+tech.readLEF("Nangate45/Nangate45_stdcell.lef")
+
+tech.readLiberty("Nangate45/Nangate45_typ.lib")
```

Comment:
You can build a tech in other ways, we just haven't enabled it yet.  You could do it entirely programmatically for example.  We could have readLEF take a list of files if that is preferrable.

Reply:
would you still be able to perform some of flow step if you had a Tech initialize with only LEF but no Liberty file?

Reply:
Yes, but that's useful to me to know that it's a valid usecase to have an incomplete `Tech` and refin it by further method calls, rather than requiring to have everything completed at construction time.

Reply:
You only need .lib if you plan to do timing.  If you just want to see or manipulate layout you don't need the .lib.  What is a 'complete' Tech is use model dependent.